### PR TITLE
Sync paratext

### PR DESCRIPTION
**Pull request by vipinpaul**
Paratext sync got broken, worked on the issue and the sync is working and tested it.

### CODE REVIEW

**Comment by joelthe1** [comment]
Are we done with testing these changes?

**Comment by vipinpaul** [comment]
Yeah, We have done the testing. As we have planned we will test it before release.
So now we need to built and test it after merging the code.

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
 BROWSER=none
+REACT_APP_PARATEXT_TOKEN=eyJhbGciOiJFUzI1NiJ9.eyJzY29wZXMiOlsib2F1dGg6cGFzc3dvcmQiXSwianRpIjoiWkFzcTd0NkRwZndmc0FIOEgiLCJhdWQiOlsiaHR0cHM6Ly9yZWdpc3RyeS5wYXJhdGV4dC5vcmciXSwicHJpbWFyeV9vcmdfaWQiOiJmNTQ3YTJmMzU2MWQ3ZGEyZGMyNzZmOWQiLCJzdWIiOiJhMnh2cGt2V3BCc2RRTkFkdCIsImF6cCI6ImEyeHZwa3ZXcEJzZFFOQWR0IiwiaWF0IjoxNTMwMTY0MDMyLCJpc3MiOiJwdHJlZyJ9.90Gor2h-JehL_BFP6Vt25AlSt-Gu5sR0QgINUJ17MttZSGagQu27PKI5LYbMlU2pkyOepB2X238OHY9ytW8Cpg
```
Comment (joelthe1) [inline .env]: Is this supposed to committed on Git? I thought this was only put in while we built the app?

Comment (vipinpaul) [reply .env]: Yeah, we don't need this. I made a new branch to avoid env file.
Will update the branch.

**Comment by vipinpaul** [comment]
Made a different branch with needed files.

**Pull request closed without merging**